gamesession.js

```diff
@@ -0,0 +1,46 @@
+const Scoreboard = require("./scoreboard.js");
+
+module.exports = class GameSession {
+    constructor() {
+        this._song = null;
+        this._artist = null;
+        this._link = null;
+        this._inSession = false;
+        this.scoreboard = new Scoreboard();
```

Comment:
We need to access its member functions, and I think it would be counter-intuitive to create more interface functions in GameSession when we can just use Scoreboard's interface functions directly.